src/wp-includes/media.php

```diff
@@ -5666,11 +5666,12 @@ function wp_get_loading_optimization_attributes( $tag_name, $attr, $context ) {
 		/*
 		 * Skip programmatically created images within post content as they need to be handled together with the other
```

Comment:
@felixarntz I was wondering is there a better word of phrase for `programmatically created images`. 
Or a line to explain what it means. This bug might have never occurred had I had a better understanding of the word programmatically created images at the first instance. I was not considering something as simple as image inside shortcode to be a progamatically created image.

Reply:
That's a fair point. I'm not sure what a better explanation would be. I consider "programmatically created images" any image that doesn't come from the database content, whether it's in a template or shortcode or block, all of those typically call a function like e.g. `wp_get_attachment_image()`. But that's not the only function, so it would also be wrong to say "any images rendered by `wp_get_attachment_image()` - although that may still be easier to understand in a way. 🤔 

If we can find a better way to phrase this, I'm happy to update.